### PR TITLE
Add aboutme file

### PR DESCRIPTION
**Pull request by NuelBytes**
## Task: `my first pull request`
- `a brief description about.emma`

### CODE REVIEW

**Comment by ToluAkin** [comment]
@NuelBytes Well done! You would have to close this pull request and open another because you are not using the right pull request title. Instead of `Add aboutme file` as your pull request title, you'd use` [1] My first pull request`. Adding the id  `[1]` is very important do as to track your tasks.

File: submissions/nellyMofya/aboutMe/aboutme.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-fintech
\ No newline at end of file
+<<<<<<< HEAD
+I seek to effectively design websites and applications. Use my coding knowledge to improve FinTech in Africa. 
+=======
```
Comment (ToluAkin) [inline submissions/nellyMofya/aboutMe/aboutme.txt:3]: Remove line 3-5 of this file.

File: submissions/nellyMofya/aboutMe/aboutme.txt [hunk 2]
```diff
@@ -1 +1,5 @@
-fintech
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (ToluAkin) [inline submissions/nellyMofya/aboutMe/aboutme.txt:1]: Remove line 1 of this file @NuelBytes

**Comment by NuelBytes** [comment]
Ohk sir

**Comment by ToluAkin** [comment]
> @NuelBytes Well done! You would have to close this pull request and open another because you are not using the right pull request title. Instead of `Add aboutme file` as your pull request title, you'd use` [1] My first pull request`. Adding the id  `[1]` is very important do as to track your tasks.

@NuelBytes Do this.

**Comment by ToluAkin** [comment]
The right pull request has been created.

**Pull request closed without merging**